colossalai/tensor/_ops/addmm.py

```diff
@@ -1,64 +1,67 @@
+from sqlalchemy import func
```

Comment:
It seems like a auto completion error...

Reply:
removed

Reply:
https://segmentfault.com/a/1190000041428523 for reference